Dockerfile

```diff
@@ -1,11 +1,20 @@
 FROM elixir:1.8
 
-ENV MIX_ENV prod
-
-ADD . /app
 WORKDIR /app
+
+ADD mix.exs /app
```

Comment:
Why are these being added?

Reply:
these are the gemfiles of the elixir world, i want to build these once before i add my file system changes `ADD . /app` otherwise i have to rebuild my deps each time i change a file - which sux when i'm live coding in docker.